Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -264,57 +265,17 @@ extension TokenSyntax: ExpressibleAsTokenSyntax {
 
  %   end
  % end
-
-% for protocol, conformances in SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES.items():
-%   if 'ExpressibleAsConditionElementList' in conformances:
-extension ${protocol} {
-  public func createConditionElementList() -> ConditionElementList {
-    ConditionElementList([self])
-  }
-}
-
-%   end
-% end
-
 % for protocol, conformances in SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES.items():
-%   if 'ConditionElement' in conformances:
+%   for conformance in conformances:
+%     node = NODE_MAP.get(conformance)
+%     if node and node.children:
+%       param = filter(lambda x : x.type_name != "TokenSyntax", node.children)[0].swift_name
```

Comment:
I don’t think this is correct. We want to the first child that doesn’t have a default value. Currently optional children also have a default value of `nil` and `TokenSyntax` children might not have a default value if they don’t have a default text (see https://github.com/apple/swift/blob/17f1cf92973d2c1b3d5fe6b70f4a33771a303e3a/utils/gyb_syntax_support/kinds.py#L58-L64)

I think a better implementation would be 

```python
non_defaulted_params = filter(lambda child : syntax_buildable_default_init_value(child, SYNTAX_TOKEN_MAP.get(child.syntax_kind)) == None, node.children)
assert len(non_defaulted_params) == 1, "ExpressibleAs conformances expects the conforming type to have an initializer with a single non-optional child"
param = non_defaulted_params[0].swift_name
```

Reply:
Ah, I didn't see that. 
Added. Tough changes the `== None` to `== ""` as we return an empty string.